python/tskit/trees.py

```diff
@@ -1608,6 +1613,21 @@ def draw_svg(
         :param bool force_root_branch: If ``True`` always plot a branch (edge) above the
             root(s) in the tree. If ``None`` (default) then only plot such root branches
             if there is a mutation above a root of the tree.
+        :param float symbol_size: Change the default size of the node and mutation
+            plotting symbols. If ``None`` (default) use a standard, sensible size.
+        :param str x_label: If given, add a labelled X axis to the plot, showing the
```

Comment:
I'm uncomfortable with using the ``label`` args as the way of turning on and off the axes. Reading code like 
```python
tree.draw_svg(x_label="", y_label="") 
```
it won't be obvious that this is to draw axes with no labels. I think it might be better to have independent ``x_axis=False``, ``y_axis=False`` arguments, even though it makes code a bit more verbose. That way we can give default axis labels ("Time" and "Genome", say), which would probably be what you want most of the time, and lead to more obvious code.

Reply:
Agree here.